### PR TITLE
Handle default trap group and remove vlan members

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: syncd/syncd.cpp [hunk 1]
```diff
@@ -846,7 +846,7 @@ swss::Logger::Priority redisGetLogLevel()
     auto plevel = g_redisClient->get("LOGLEVEL");
 
     if (plevel == NULL)
-        return swss::Logger::SWSS_NOTICE;
+        return swss::Logger::SWSS_DEBUG;
```
Comment (stcheng) [inline syncd/syncd.cpp]: keep NOTICE

File: syncd/syncd.cpp [hunk 2]
```diff
@@ -1250,7 +1250,7 @@ bool isVeryFirstRun()
 
 int main(int argc, char **argv)
 {
-    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_NOTICE);
+    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_DEBUG);
```
Comment (stcheng) [inline syncd/syncd.cpp]: keep NOTICE

**Comment by stcheng** [comment]
let's do some basic test before merging

File: syncd/syncd_hard_reinit.cpp [hunk 3]
```diff
@@ -157,10 +157,56 @@ void checkAllIds()
...
+    vlanMemberList.resize(portList.size());
+
+    sai_attribute_t attr;
+
+    attr.id = SAI_VLAN_ATTR_MEMBER_LIST;
+
+    attr.value.objlist.count = vlanMemberList.size();
+    attr.value.objlist.list = vlanMemberList.data();
+
+    sai_status_t status = sai_vlan_api->get_vlan_attribute(DEFAULT_VLAN_NUMBER, 1, &attr);
```
Comment (kcudnik) [inline syncd/syncd_hard_reinit.cpp]: than it still should return zero members on list?

File: syncd/syncd.cpp (hunk 1 above)
Comment (kcudnik) [inline syncd/syncd.cpp]: fixed

File: syncd/syncd.cpp (hunk 2 above)
Comment (kcudnik) [inline syncd/syncd.cpp]: fixed

File: syncd/syncd_hard_reinit.cpp (hunk 3 above)
Comment (lguohan) [inline syncd/syncd_hard_reinit.cpp]: then you need resize the vlanMemberList otherwise, in line 191 it enumernate many members.

Comment (stcheng) [inline syncd/syncd_hard_reinit.cpp]: +1 to resize again

Comment (kcudnik) [inline syncd/syncd_hard_reinit.cpp]: ah yes

**Pull request merged**